testsuite/tests/bsc-991252.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+
+# testedfiles: helper.rb
+
```

Comment:
I have just one comment here: `bsc-991252` is a great reference, but as the history shows, we often fix something which was actually not reported, because the root of the issue was something else. So, it would make sense to put some comment/information into the actual test that describes the scenario and test-case itself in human-readable form and explains what is the test about.

I like the way you create these tests for bugs, that's a good idea. I'm just looking into the future - how to benefit from that even more. Thanks!